jax/_src/numpy/lax_numpy.py

```diff
@@ -1704,14 +1704,19 @@ def _where(condition, x=None, y=None):
 At present, JAX does not support JIT-compilation of the single-argument form
 of :py:func:`jax.numpy.where` because its output shape is data-dependent. The
 three-argument form does not have a data-dependent shape and can be JIT-compiled
-successfully.
+successfully. Alternatively, you can specify the optional ``size`` keyword.
+If specified, the first ``size`` True elements will be returned; if there
+are fewer True elements than ``size`` indicates, the index arrays will be
+padded with zeros.
```

Comment:
Small suggestion - link the last sentences maybe? They may look more cohesive if together? 🤷‍♂️ 

```suggestion
successfully. Alternatively, you can specify the optional ``size`` keyword:
if specified, the first ``size`` True elements will be returned; and if there
are fewer True elements than ``size`` indicates, the index arrays will be
padded with zeros.
```

Reply:
Done, thanks